pyrobuf/parse_proto.py

```diff
@@ -331,51 +332,114 @@ def add_cython_info(self, message):
             self.add_cython_info(submessage)
 
 
-class ParserOption(object):
-    def __init__(self, pos, option):
-        self.token_type = 'OPTION'
-        self.pos = pos
-        self.option = option
-
-
-class ParserSyntax(object):
-    def __init__(self, pos, value):
-        self.token_type = 'SYNTAX'
-        self.pos = pos
-        self.value = value
-
-
-class ParserImport(object):
-    def __init__(self, pos, value):
-        self.token_type = 'IMPORT'
-        self.pos = pos
-        self.value = value
-
+    class ParserOption(object):
+        def __init__(self, pos, option):
+            self.token_type = 'OPTION'
+            self.pos = pos
+            self.option = option
+
```

Comment:
Think we should make this just a single space between classes now that they are children of a parent class.

Reply:
Fixed :+1: